tests/flows/check_upgrade_access.go

```diff
@@ -62,8 +67,13 @@ func CheckUpgradeAccess(network interfaces.Network) {
 	// Try to call pauseTeleporterAddress from the owner account
 	tx, err := exampleMessenger.PauseTeleporterAddress(ownerOpts, teleporterAddress)
 	Expect(err).Should(BeNil())
-	utils.WaitForTransactionSuccess(ctx, subnetInfo, tx)
-	isPaused, err = exampleMessenger.IsTeleporterAddressPaused(&bind.CallOpts{}, teleporterAddress)
+	receipt := utils.WaitForTransactionSuccess(ctx, subnetInfo, tx)
+	blockNum := receipt.BlockNumber
+	pauseTeleporterEvent, err := utils.GetEventFromLogs(receipt.Logs, exampleMessenger.ParseTeleporterAddressPaused)
```

Comment:
Can you leave a comment explaining why we should not provide an empty `bind.Callopts` like we do elsewhere in the e2e tests?